package.json

```diff
@@ -47,28 +46,31 @@
     "lodash.debounce": "^4.0.8",
     "log4js": "^6.3.0",
     "node-fetch": "^2.6.1",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0"
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
   },
   "devDependencies": {
     "@aws-cdk/aws-apigateway": "^1.22.0",
     "@aws-cdk/aws-apigatewayv2": "^1.22.0",
     "@aws-cdk/aws-lambda": "^1.22.0",
     "@aws-cdk/core": "^1.22.0",
-    "@babel/core": "^7.8.3",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/plugin-proposal-numeric-separator": "^7.10.4",
-    "@babel/plugin-proposal-optional-chaining": "^7.10.4",
-    "@babel/preset-env": "^7.10.4",
-    "@babel/preset-typescript": "^7.10.4",
+    "@babel/core": "^7.13.15",
+    "@babel/plugin-proposal-class-properties": "^7.13.0",
+    "@babel/plugin-proposal-numeric-separator": "^7.12.13",
+    "@babel/plugin-proposal-optional-chaining": "^7.13.12",
+    "@babel/plugin-transform-react-jsx-compat": "^7.12.13",
+    "@babel/preset-env": "^7.13.15",
+    "@babel/preset-react": "^7.13.13",
+    "@babel/preset-typescript": "^7.13.0",
+    "@babel/standalone": "^7.13.15",
```

Comment:
there does not seem to be any major version bump in @babal packages, so unlikely for issues to come from there